pkg/wallet/wallet_list_actions_test.go

```diff
@@ -104,10 +96,11 @@ func (s *WalletTestSuite) TestWalletListActions() {
 		// then:
 		assert.NoError(t, err)
 		require.NotNil(t, result)
-		assert.GreaterOrEqual(t, int(result.TotalActions), 0, "Should have non-negative total actions")
+		assert.Equal(t, uint32(0), result.TotalActions, "Should have zero total actions")
+		assert.Empty(t, result.Actions, "Actions list should be empty")
 	})
 
-	s.Run("list actions with all includes", func() {
+	s.Run("list actions after create action", func() {
```

Comment:
this isn't what I mean 🙈 

There are already tests for CreateAction, just add another `when:` `then:` section there that will check the list actions result after that call